### PR TITLE
Update Blog “okta-sso-integration-for-green-lake-and-aruba-central”

### PR DESCRIPTION
**Pull request by WifiGuyWill**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 1]
```diff
@@ -59,25 +59,25 @@ Provide a name for the Aruba GreenLake SSO service (Okta application)
 
     **NameID = user.email**
 
-    **gl\_first\_name = user.FirstName**
+    **gl_first_name = user.FirstName**
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:62]: Some characters (underscore, hash, backslash) have special meanings in the Markdown syntax. If you want to use these special characters in a text, you have to escape them. The way to escape a special character is to add a backslash (\\) before it, for example: I do not want \_italic text\_ here.

Therefore, throughout the blog post, make sure to use the backslash in front of the "underscore" character for the variables such as gl_first_name.

You can check the preview of the updated blog post here:
https://deploy-preview-2421--hpe-dev-portal.netlify.app/blog/okta-sso-integration-for-green-lake-and-aruba-central/

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 2]
```diff
@@ -59,25 +59,25 @@ Provide a name for the Aruba GreenLake SSO service (Okta application)
...
 
-    **gl\_last\_name = user.LastName**
+    **gl_last_name = user.LastName**
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:64]: Escape the underscore in the variable.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 3]
```diff
@@ -59,25 +59,25 @@ Provide a name for the Aruba GreenLake SSO service (Okta application)
...
 
-    **hpe\_ccs\_attribute = (See Below)**
+    **hpe_ccs_attribute = (See Below)**
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:66]: Escape the underscore in the variable.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 4]
```diff
@@ -59,25 +59,25 @@ Provide a name for the Aruba GreenLake SSO service (Okta application)
...
 
    See here for IdP attribute details: <https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us>
 
-   As part of the HPE GreenLake cloud platform integration, one of the additional features that was added is the Role Based Access Controls for Aruba Central and all other apps on the platform. A new SAML attribute has been added “hpe\_ccs\_attribute” which tells HPE GreenLake and Central the exact role/permissions for each user. The following describes how to format the attribute.
+   As part of the HPE GreenLake cloud platform integration, one of the additional features that was added is the Role Based Access Controls for Aruba Central and all other apps on the platform. A new SAML attribute has been added “hpe_ccs_attribute” which tells HPE GreenLake and Central the exact role/permissions for each user. The following describes how to format the attribute.
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:70]: Escape the underscore for "hpe_ccs_attribute".

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 5]
```diff
@@ -59,25 +59,25 @@ Provide a name for the Aruba GreenLake SSO service (Okta application)
...
 
 ![](/img/ws-image3.png)
 
-![](/img/ws-image4.png)
+![](/img/ws-image17.png)
 
 ![](/img/ws-image5.png)
 
-The **hpe\_ccs\_attribute** always starts with version_1#. You must first configure the attributes for HPE GreenLake CSS, and then Central. To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and **ALL_SCOPES**. Next, enter in the Aruba Central information. Start with the **app cid**, followed by the role name (i.e. Aruba Central Administrator), and then **ALL_SCOPES**.
+The **hpe_ccs_attribute** always starts with version_1#. You must first configure the attributes for HPE GreenLake CSS, and then Central. To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and **ALL_SCOPES**. Next, enter in the Aruba Central information. Start with the **app cid**, followed by the role name (i.e. Aruba Central Administrator), and then **ALL_SCOPES**.
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:80]: Escape the underscore for "hpe_ccs_attribute", for "version_1#" and for "ALL_SCOPES" and in the "example that follows this paragraph:

Example:

**version1#** 5b0ec0e8c4f422eca232ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:**ALLSCOPES**:683da368-66cb-4ee7-90a9-ec1964768092:

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 6]
```diff
@@ -154,13 +145,7 @@ Enter the SSO credentials. You will be redirected to Okta to authenticate. Once
 **Additional notes:**
 
 * There must be at least **one** verified user belonging to the **Domain** prior to configuration.    
-
 * In order to configure SSO, you must be logged into HPE GreenLake with a user from the domain.    
-
 * SSO user access is determined by the “role_name” attribute included in the SAML hpe_ccs_attribute provided by the IdP.
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:149]: Escape the underscore for "role_name" and "hpe_ccs_attribute"

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Will,

Thank you very much for keeping the content of your blog post current and relevant.
I have added a few comments to make sure you escape the "underscore" character in the variables throughout the blog post. Please check my comments. 

You can use the preview of the blog post to check whether the underscore characters are correctly escaped for the variables like "hpe_css_attribute", "role_name", "ALL_SCOPES", "version_1#".

https://deploy-preview-2421--hpe-dev-portal.netlify.app/blog/okta-sso-integration-for-green-lake-and-aruba-central/

If it makes sense to you, you can also update the date of the blog post to a more recent date.

Thanks again
Denis

**Comment by WifiGuyWill** [comment]
@choukden Thanks for suggestions, I added the escape for the underscore. I just realized I probably didn't need to move it back to drafts to make those changes. Sorry for any confusion.

**Comment by choukden** [comment]
@WifiGuyWill 
Thank you very much for taking into account my comments.
Exactly, you do not need to move the blog post from "In Review" to "Drafts" in the Workflow section of the CMS to make changes. No worry.

Let me know if you want me to push the updated blog post to production (that is, to the HPE Dev Community portal).

Thanks
Denis

**Comment by WifiGuyWill** [comment]
@choukden 
I fixed a couple more minor formatting issues. The post is good and can be pushed to production. Thanks again for your help.

**Pull request closed without merging**